site/docs/components/side_navigation.mdx

```diff
@@ -0,0 +1,101 @@
+import Link from "../../src/components/Link";
+
+# Side navigation
+
+<StatusLabel type="alert">Draft</StatusLabel>
+<StatusLabel type="success" style={{marginLeft: 'var(--spacing-xs)'}}>Accessible</StatusLabel>
+
+<LargeParagraph>
+  A side navigation is an additional navigation element which is located at the left-hand side of the viewport. It provides an extra level of navigation and also can be helpful in situations where the traditional top navigation feels too clumsy.
+</LargeParagraph>
+
+## Principles
+
+- **It is strongly recommended to always include the [main (top) navigation component](/components/navigation) in your service.**
+  - It is allowed to use the side navigation without the top navigation. However, please note that the top navigation includes important features such as language selection and login. If you rely on the side navigation only, you need to provide these functionalities somewhere else in the user interface.
+- If you use both main (top) and side navigation, do not repeat same navigation items in both of them.
+- Always position the side navigation to the left-hand side of the page. Keep the navigation in place so the user can always easily locate it.
+- Keep navigation menu link labels clear and concise. Prefer max 1-2 word long labels. Avoid starting multiple menu link labels with the same word.
+- The side navigation allows the use of icons for the first level links.
+  - If icons are used, all first level links should have an icon.
+  - Pay attention to the icon choices. The icons should be easily recognisable and not used for decorative purposes only.
+  - The second level links and external links must not include an icon.
+  - Labels are always required even if icons are used. Do not create icon-only navigations.
+- When including external links in the side navigation, follow guidelines of the [HDS Link component documentation]().
+
+### When to use a side navigation?
+
+- **Do use the Side navigation when**
+  - the service has more than three (3) levels in its navigation hierarchy
+  - the user needs to be able to easily scan through and navigate between pages of a specific level (e.g. admin interfaces or documentation sites)
+  - you are uncertain of the number of the navigation items and you need room for growth and changes
+- **Don't use the Side navigation when**
+  - you need space for your content (the side navigation uses a large portion of the content area)
+  - your service has a low amount of navigation items and only one level deep navigation hierarchy
+
+## Accessibility
+- **When designing a menu link hierarchy, always think about keyboard and screen reader users.** First, make sure top-level menu labels are clear, and submenu items contextually fit under it. Next, make sure the menu option order is logical and reasonable. Remember that keyboard users need to manually navigate to each element and thus placing a commonly used menu option last is not optimal.
```

Comment:
and submenu items contextually fit under it -> and submenu items contextually fit under **the top-level**.